holidays/calendars/thai.py

```diff
@@ -264,34 +266,30 @@ def __init__(self, calendar=THAI_CALENDAR) -> None:
 
     @staticmethod
     def __is_khmer_calendar(calendar):
-        """
-        Return True if `calendar` is Khmer calendar.
-        Return False otherwise.
-        """
+        """Return True if the given `calendar` is KHMER_CALENDAR; False otherwise."""
```

Comment:
```python
    def __is_khmer_calendar(calendar: str) -> bool:
        """Check if the given calendar is the Khmer calendar.

        Args:
            calendar:
                The calendar identifier to check.

        Returns:
            True if the calendar is KHMER_CALENDAR, False otherwise.
        """
        return calendar == KHMER_CALENDAR
```